.cloudbuild/deploy.yaml

```diff
@@ -19,18 +21,18 @@ steps:
   - name: 'gcr.io/cloud-builders/gcloud'
     entrypoint: 'bash'
     args:
-    - '-c'
-    - |
-      # This snippet lets us pass additional args via the Cloud Build console
-      # in the _EXTRA_GCLOUD_ARGS var.
-      # nb. We don't have to specify --project; it's part of the environment.
-      gcloud app deploy ${_EXTRA_GCLOUD_ARGS}
+      - '-c'
```

Comment:
My IDE's code formatter fixed a few things that were non-standard about the current YAML. I can revert them if needed, but it seemed harmless.